SUMMARY.md

```diff
@@ -109,6 +109,7 @@
     * [Customizing case details tabs](extending-valtimo/tabs/customizing-case-tabs.md)
     * [Setting extra tabs for Zaak object type](extending-valtimo/tabs/configure-tab-object-type.md)
     * [Customizing case list tabs](extending-valtimo/tabs/customizing-case-list-tabs.md)
+  * [Integrating spring beans in a process](extending-valtimo/integrate-spring-bean-in-process.md)
```

Comment:
Can you add the creating a custom dashboard page here as well?

Reply:
@marijnritense Am i allowed to say no? :)